src/server/lib/server.vcxproj

```diff
@@ -59,7 +59,7 @@
   </ItemGroup>
   <ItemGroup>
     <ProjectReference Include="..\..\host\proxy\Host.Proxy.vcxproj">
-      <Project>{e437b604-3e98-4f40-a927-e173e818ea4b}</Project>

```

Comment:
wait what?

Reply:
IDK man. I saved the sln in VS, and it tore `e437b604-3e98-4f40-a927-e173e818ea4b` out of the .sln. I did a Find All, and this was the only reference to that guid, and _this isn't even the GUID of `Host.Proxy`_

Reply:
Alright fine.